app/models/solidus_stripe/payment_method.rb

```diff
@@ -25,6 +25,20 @@ def gateway_class
       Gateway
     end
 
+    def create_profile(payment)
+      pi = payment.source.payment_intent
```

Comment:
I'd prefer to avoid 2-letter variables. `payment_intent` is not that long that will create problems IMHO, thoughts?